nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -12,7 +12,8 @@
 from scipy import linalg
 from sklearn.utils import check_random_state
```

Comment:
All what comes next is unrelated, right ?

Reply:
Yeah, that's weird... These changes are coming from #3131 after I updated from `main`. I'm not sure why they're showing up in the diff.